### PR TITLE
feat: add alert and button

### PR DESCRIPTION
**Pull request by shaik-saad**
Added differnt types od alert and buttons

csb for alert:
https://s60ps.csb.app/components/alert/alert.html

csb for button:
https://s60ps.csb.app/components/button/button.html

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    border-radius: 2.5px;
+}
+
+.btn-primary-outline:hover {
+    background: var(--primary-color);
+    color: var(--white-color);
+}
+.btn-success-outline {
+    border: 2px solid var(--success-color);
+    color: var();
```
Comment (ruchiragoswami) [inline components/button/button.css:50]: Please take a look here.

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    padding: 0.5rem 1rem;
+    margin: 0.5rem;
+    background: var(--white-color);
+}
+
+.btn-primary {
+    background: var(--primary-color);
+    color: var(--white-color);
+    border-radius: 2.5px;
+    /* opacity: 0.5; */
```
Comment (ruchiragoswami) [inline components/button/button.css:12]: is the comment required here?

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    border-radius: 2.5px;
+}
+
+.btn-success-outline:hover {
+    background: var(--success-color);
+    color: var(--white-color);
+}
+.btn-warning-outline {
+    border: 2px solid var(--warning-color);
+    color: var();
```
Comment (ruchiragoswami) [inline components/button/button.css:60]: Sorry, I am not sure about empty variable.

File: components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    border-radius: 2.5px;
+}
+
+.btn-warning-outline:hover {
+    background: var(--warning-color);
+    color: var(--white-color);
+}
+.btn-danger-outline {
+    border: 2px solid var(--danger-color);
+    color: var();
```
Comment (ruchiragoswami) [inline components/button/button.css:70]: Empty variables on lines 70 and 80.

**Review by ruchiragoswami — COMMENTED** [review]
Looks great! 
I asked some questions in comments. Please let me know and I will approve.

File: components/button/button.css (hunk 2 above)
Comment (shaik-saad) [reply components/button/button.css:12]: No, removed it.

File: components/button/button.css (hunk 4 above)
Comment (shaik-saad) [reply components/button/button.css:70]: Oops, I missed to add the names. Corrected them.

**Review by ruchiragoswami — APPROVED** [review]
All good now

**Pull request closed without merging**